src/ServerScriptService/MainSystem.server.lua

```diff
@@ -4,23 +4,21 @@ local Players = game:GetService("Players")
 -- Example #1: Kicking Players if Datastores are down.
 
 Players.PlayerAdded:Connect(function(player)
-    if DowntimeService:GetDatastoreAPIStatus() == "down" then
-        player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
-    end 
+	if DowntimeService.statusFromAlias("datastore")== "Down" then
+		player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
+	end 
 end)
 
--- Example #2 Printing latest tests if something is down or not. 
+-- Example #2 Printing the Roblox website's status every 5 minutes. 
 
 while true do
```

Comment:
While the code is better than before, it is most likely a better idea to use `RunService.Heartbeat` rather than `while true do`